module/x/gravity/abci.go

```diff
@@ -61,6 +61,10 @@ func createBatchTxs(ctx sdk.Context, k keeper.Keeper) {
 }
 
 func createSignerSetTxs(ctx sdk.Context, k keeper.Keeper) {
+	params := k.GetParams(ctx)
```

Comment:
If we disable the bridge and the total validator change is more than 1/3 of total power, then there is a risk that we cannot update the signerset anymore as it requires 2/3 of power of the "old" signerset

But this should not be an issue on our mainnet

Reply:
what's the plan for it if that happened? override it in the upgrade handler?

Reply:
no the issue is not on the signatures at cosmos level but on the smart contract level which expects the old valset to sign for the new valset (using their ethereum keys)

In case this happen, we can migrate to a new gravity contract and reinitialize the valset